images/prometheus-adapter/config/template.apko.yaml

```diff
@@ -0,0 +1,17 @@
+contents:
+  packages: [
+    "prometheus-adapter"
```

Comment:
I think we'll want this to be provided by `extra_packages` so we can control this from outside this YAML

Reply:
that makes sense, updated!